spec/models/consent_form_spec.rb

```diff
@@ -738,9 +738,33 @@
     expect(consent_form.health_answers).not_to be_empty
   end
 
-  it "removes the health questions when the parent refuses consent" do
```

Comment:
This spec started failing when the feature flag was removed; I'm a bit unclear how it was passing previously when the session (which is randomly flu or hpv, if I'm reading the factory code correctly) was a flu session.

I've split the spec so there is one for each type of programme.

Reply:
Ah, this spec is flaky for me on `main`.